src/ol_infrastructure/substructure/keycloak/Pulumi.substructure.keycloak.Production.yaml

```diff
@@ -2,13 +2,18 @@
 secretsprovider: awskms://alias/infrastructure-secrets-production
 encryptedkey: AQICAHiiGjYUolrtj8PCnScLM7oLAdMl8nJrLjQjnqyl1LykYgF6LD3AC9aDhHZ8J3SdTKTrAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM7UqV4xkz6mym2LbAAgEQgDt2BFqcsnYAUr+fyvsAmQKTBuYpEHcraW8xU6+UrD/++MwqrkxANEP5lwmE4EnM+xuIb7KntcacHB9tgw==
 config:
+  keycloak:client_id: pulumi
+  keycloak:client_secret:
+    secure: v1:mbdWdrtlOrf/yMPw:4cu91sZzAKkYUkyxQBEtClJ49/uRp/f9nuW4uDakdRV7RW57Yf1EIqd71ovo0+TZ
+  keycloak:mailgun_email_password:
+    secure: v1:FSbXwQ1CxlTHTdYx:6MeC7MT3fTFR52qVTBxXBf6Ic2WbwCAxjWQSgr4Sf8GB0zfYtFJpGi7mK54idUivKC7H4qJoMX8xt30SOBP8stbU
+  keycloak:mailgun_email_username:
+    secure: v1:gtHzgnPCwsxt+URU:Z6jI+P2sao5cVwHcZ0TSbJsO/HAqJ6k8GDjXCN5iqUsV5EGst0eXX+Y2c3Q=
```

Comment:
I think these configuration values need to be scoped under a different namespace, otherwise the Keycloak provider will start complaining about not knowing what those settings mean. That's why I added the other values as `keycloak_realm`